requirements-dev.txt

```diff
@@ -2,3 +2,4 @@ pytest==3.1.3
 pytest-cov==2.5.1
 -r requirements-docs.txt
 wheel==0.33.6
+psycopg2==2.7.7
```

Comment:
Why this old version of psycopg2 specifically ?

Reply:
the existing tests fail due to a parameter change with psycopg2 2.8.x